tests/test_mechanics.py

```diff
@@ -279,6 +279,31 @@ def test_discard_enchanted_cards():
 	assert not game.player1.hand
 
 
+def test_discover():
```

Comment:
Note that for AI purposes we need to generate all possible outcomes for a given selection, with a table of probabilities, so I will be ensuring a good API is available for this :)